cluster/gce/gci/configure-helper.sh

```diff
@@ -1701,17 +1701,43 @@ function prepare-etcd-manifest {
   if [[ -n "${INITIAL_ETCD_CLUSTER_STATE:-}" ]]; then
     cluster_state="${INITIAL_ETCD_CLUSTER_STATE}"
   fi
+
+  # Configure mTLS for etcd peers.
   if [[ -n "${ETCD_CA_CERT:-}" && -n "${ETCD_PEER_KEY:-}" && -n "${ETCD_PEER_CERT:-}" ]]; then
     etcd_creds=" --peer-trusted-ca-file /etc/srv/kubernetes/etcd-ca.crt --peer-cert-file /etc/srv/kubernetes/etcd-peer.crt --peer-key-file /etc/srv/kubernetes/etcd-peer.key -peer-client-cert-auth "
     etcd_protocol="https"
   fi
 
-  # mTLS should only be enabled for etcd server but not etcd-events. if $1 suffix is empty, it's etcd server.
+  # In a zonal cluster, etcd_host_ip and host_ip are both the same as host VM's internal IP.
+  # In a regional cluster, a secondary alias IP range (usually 10.127.X.X) is allocated for etcd servers.
+  local host_ip=""
```

Comment:
Curious why is this empty by default, v.s apiserver uses 127.0.0.1 as default?

Reply:
Oh good suggestion. I left it empty because it won't be used if the feature flag is not true. But I guess it's just cleaner to default to a value.